src/Molecules/LeaderboardList/LeaderboardList.js

```diff
@@ -0,0 +1,25 @@
+import LeaderboardListElement from './../../Atoms/Leaderboard/LeaderboardListElement';
+import LeaderboardListStyled from './LeaderboardListStyled';
+import propTypes from 'prop-types';
+
+const LeaderboardList = ({ result }) => {
+  return (
+    <LeaderboardListStyled>
+      {result.map((arrElement, index) => {
+        return (
+          <LeaderboardListElement
+            key={index}
+            text={arrElement.text}
+            score={arrElement.score}
+            index={index}
+          />
+        );
+      })}
+    </LeaderboardListStyled>
+  );
+};
+
+LeaderboardList.propTypes = {
```

Comment:
should be required?

Reply:
Nothing will run without the object input, so yes :)